packages/tokens/src/tokens/font/composite.json

```diff
@@ -0,0 +1,152 @@
+{
+  "vads-font-display": {
```

Comment:
Just adding a note that mostly deferring to Jessica for this file. General format looks fine, did not validate values.